astropy/io/fits/file.py

```diff
@@ -665,9 +658,7 @@ def _mmap_available(cls):
                 mm = mmap.mmap(tmpfd, 1, access=mmap.ACCESS_WRITE)
             except OSError as exc:
                 warnings.warn(
-                    "Failed to create mmap: {}; mmap use will be disabled".format(
-                        str(exc)
-                    ),
+                    f"Failed to create mmap: {str(exc)}; mmap use will be disabled",
```

Comment:
`str()` will be called implicitly here so can be removed.

Reply:
Done